zeroeq/detail/fbevent.h

```diff
@@ -45,10 +42,14 @@ class Event
         return parser.builder_.GetBufferPointer();
     }
 
-    void setData( const ConstByteArray& data_, const size_t size_ )
+    void setData( const uint8_t* data_, const size_t size_ )
     {
         parser.builder_.Clear();
-        data = data_;
+        // Since we are working on the same copy with new servus::Serializable
+        // based Flat Buffer events ( not memory per event ),
+        // we are copying the memory
+        data.reset( new uint8_t[ size_ ] );
+        memcpy( data.get(), data_, size_ );
```

Comment:
This copy is not needed. The contract is that the memory is valid only in the updated func, which should be (is not right now) called while the zmq msg is still open.